CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ Changelog for NeoFS Node
 - Support of `GT`, `GE`, `LT` and `LE` numeric comparison operators in CLI (#2733)
 - SN eACL processing of NULL and numeric operators (#2742)
 - CLI now allows to create and print eACL with numeric filters (#2742)
-- gRPC connection limits per endpoint (#1240) 
+- gRPC connection limits per endpoint (#1240)
```

Comment:
any change here? GH doesnt mark any on my side

Reply:
redundant space at the end

Reply:
Usually they can be seen in GH diff.

Reply:
> redundant space

oh rly

> Usually they can be seen in GH diff.

seems like additional line makes it invisible in diff. Being a separate commit, it'd be visible at least in `Commits` (not in `Files changes` i suppose)

Reply:
> Usually they can be seen in GH diff.

i guess that is caused by the moving `\n` to the next line

> Being a separate commit, it'd be visible at least in Commits

IMO, a separate commit is too much. i know we love commits but that is moving nonvisible symbols to another place because of a new line

Reply:
im not callin for a separate commit (although independent fix in fact), just tried to imagine and understand why it was visible in diff of past PRs